code/smartPointers/smartPointers.cpp

```diff
@@ -252,7 +252,7 @@ class Owner {
 void problem4_1() {
     std::vector<Owner> owners;
 
-    for (unsigned int i=0; i < 5; ++i) {
+    for (int i=0; i < 5; ++i) {
```

Comment:
Changed to `int` because you compare `i < 5`, where the second operand is also an `int`. But honestly, it does not matter :) `i` is also not used in the body of the loop.